opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -243,6 +243,8 @@ namespace Opm {
                 residual_norms_history_.clear();
                 current_relaxation_ = 1.0;
                 dx_old_ = 0.0;
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
```

Comment:
IMO, that input conversion stuff caused more than its fair share of trouble by now. That said, changing it is a substantial amount of work, so for it's for another day...